Update skipping-index.md

lesandie opened the pull request:
DROP is not such lightweight command imo it generates  a mutations to deal with the changes and they will take time to complete depending on how much data the table has.

### Changelog category (leave one):
- Documentation (changelog entry is not required)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

lesandie committed:
Update skipping-index.md

ADD and DROP are not such lightweight command they generate mutations to deal with the changes and they will take time to complete depending on how much data the table has.

lesandie committed:
Merge branch 'master' into patch-1

alexey-milovidov commented:
> will add/drop a column for the index for each part

This statement is confusing. Maybe you meant "a file"? But the ADD INDEX does not add any files.

lesandie committed:
Merge branch 'ClickHouse:master' into patch-1

lesandie commented:
> > will add/drop a column for the index for each part
> 
> This statement is confusing. Maybe you meant "a file"? But the ADD INDEX does not add any files.

Yes you're correct, let me refrase and correct it

lesandie committed:
Update skipping-index.md

Deleted the confusing comment and added implemented as a mutation, like MATERIALIZE INDEX.
